File: app/src/main/res/ui/splash/SplashActivity.java
package com.khedr_and_basuoin.deliveryapp.ui.splash;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.app.AppCompatDelegate;
import androidx.databinding.DataBindingUtil;

import com.khedr_and_basuoin.deliveryapp.R;
import com.khedr_and_basuoin.deliveryapp.databinding.ActivitySplashBinding;
import com.khedr_and_basuoin.deliveryapp.ui.welcome.WelcomeActivity;

public class SplashActivity extends AppCompatActivity {
    ActivitySplashBinding b;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_NO);
        super.onCreate(savedInstanceState);

        b= DataBindingUtil.setContentView(this, R.layout.activity_splash);
        new Handler().postDelayed(() -> {
            startActivity(new Intent(this, WelcomeActivity.class));
            finish();
        },3500);

    }
}

File: app/src/main/java/com/khedrandbasuoni/baytifood/ui/activities/splash/SplashActivity.kt
package com.khedrandbasuoni.baytifood.ui.activities.splash

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.appcompat.app.AppCompatDelegate
import androidx.databinding.DataBindingUtil
import com.khedrandbasuoni.baytifood.R
import android.content.Intent
import android.os.Handler
import android.os.Looper
import com.khedrandbasuoni.baytifood.databinding.ActivitySplashBinding
import com.khedrandbasuoni.baytifood.ui.activities.welcome.WelcomeActivity

class SplashActivity : AppCompatActivity() {
    lateinit var b: ActivitySplashBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_NO)
        super.onCreate(savedInstanceState)
        b = DataBindingUtil.setContentView(this, R.layout.activity_splash)
        Handler(Looper.getMainLooper()).postDelayed({
            startActivity(Intent(this, WelcomeActivity::class.java))
            finish()
        }, 3500)
    }
}